android/AndroidManifest.xml

```diff
@@ -62,4 +62,8 @@
          Remove the comment if you do not require these default features. -->
     <!-- %%INSERT_FEATURES -->
 
+<!-- %%INSERT_PERMISSIONS -->
```

Comment:
Please see commit a6e17fb6bee6c150c30959cbe03b5ede075d917e:
```
[android] Explicitly set permissions

Otherwise Qt will fill the permissions in during build time and add to
the ones listed below:

- android.permission.CAMERA
- android.permission.RECORD_AUDIO

which we don't need and only confuses the user.
```

Reply:
Very true. Thank you! Unfortunately, it is already released like this :/
Next time!